Source/Controllers/MainViewControllers/ConnectionView/SPConnectionController.m

```diff
@@ -736,13 +736,13 @@ - (IBAction)updateKeyLocationFileVisibility:(id)sender
  */
 - (void)updateSplitViewSize
 {
-	if ([dbDocument getConnection]) {
-		return;
-	}
-
-	[connectionSplitView setDelegate:nil];
-	[connectionSplitView setPosition:[[[databaseConnectionView subviews] firstObject] frame].size.width ofDividerAtIndex:0];
-	[connectionSplitView setDelegate:self];
+//	if ([dbDocument getConnection]) {
+//		return;
+//	}
+//
+//	[connectionSplitView setDelegate:nil];
+//	[connectionSplitView setPosition:[[[databaseConnectionView subviews] firstObject] frame].size.width ofDividerAtIndex:0];
+//	[connectionSplitView setDelegate:self];
 }
```

Comment:
I believe this was a @jamesstout fix for some earlier bug that ya commented out here (and in the splitViewDidResizeSubviews block below). Why the comment out?

Reply:
Nah the fix was with nil coordinates in SPAppController:

```
NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];

        // fake the dbViewInfoPanelSplit being open
        NSMutableArray *dbViewInfoPanelSplit = [[NSMutableArray alloc] initWithCapacity:2];
        [dbViewInfoPanelSplit addObject:@"0.000000, 0.000000, 359.500000, 577.500000, NO, NO"];
        [dbViewInfoPanelSplit addObject:@"0.000000, 586.500000, 359.500000, 190.500000, NO, NO"];
        [prefs setObject:dbViewInfoPanelSplit forKey:@"NSSplitView Subview Frames DbViewInfoPanelSplit"];
```

Removing this fixes an issue where every new tab has width `2*leftPane.width` of the previousTab